services/user.js

```diff
@@ -5,10 +5,21 @@ const { createError } = require('~/utils/errorsHelper')
 const { USER_NOT_FOUND, ALREADY_REGISTERED } = require('~/consts/errors')
 
 const userService = {
-  getUsers: async () => {
-    const users = await User.find().populate('categories').select('-__v').lean().exec()
+  getUsers: async ({ match, sort, skip, limit }) => {
+    const count = await User.countDocuments(match)
+
+    const items = await User.find(match)
+      .select('+isEmailConfirmed +isFirstLogin')
```

Comment:
if you need those fields for all users then just remove select: false from the model for those fileds